news/2022-04-02-new-featured-artist-a-saka.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "New Featured Artist: A.SAKA"
+date: 2022-04-02 09:00:00 +0000
+---
+
+**A.SAKA** whirlwinds their way into our Featured Artist collection!
+
+![](https://assets.ppy.sh/artists/253/header.jpg)
+
+Specializing in Japanese-style instrumental tunes that translate perfectly to rhythm games, **A.SAKA**'s sound has unsurprisingly been a hit on osu! since day 1. With multiple beatmaps of their most popular track [Nanatsu Issenzakura](https://osu.ppy.sh/beatmapsets?q=%22Nanatsu%20Issenzakura%22) reaching over a million plays, there's a good chance you're already familiar with **A.SAKA**'s legendary sound.
+
+We've scooped up **12** songs for [**A.SAKA**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/253) (obviously including *Nanatsu Issenzakura*).
+
+If you're one of the four osu! players who somehow *hasn't* heard this timeless classic, shift your attention towards this multi-mode gameplay video, then peek at the rest of **A.SAKA**'s awesome track previews below:
```

Comment:
```suggestion
If you're one of the four osu! players who somehow *haven't* heard this timeless classic, shift your attention towards this multi-mode gameplay video, then peek at the rest of **A.SAKA**'s awesome track previews below:
```

Reply:
the original is correct grammar :(

Reply:
it's either "the four osu! players" or "you", both of which take "have"

Reply:
I'm not even sure anymore. "you're one" that "hasn't" seems correct after all.

Reply:
Let's go with original, it flows better off the tongue when reading out loud.

Reply:
hmm nvm, you're right, but it can go either way depending on how you put it: "has" if it refers to "one" or "have" if it refers to "the four osu! players". I'll rest my case in that the latter is closer and makes more sense